README.rst

```diff
@@ -137,13 +151,57 @@ You can use it with django-templated-email, just follow their instructions in th
 
 Optionally you can use their custom `EmailMessage <https://anymail.readthedocs.io/en/latest/sending/anymail_additions/#anymail.message.AnymailMessage>`_ class with django-templated-email by using the following settings:
 
+.. code-block:: python
+
     # This replaces django.core.mail.EmailMessage
     TEMPLATED_EMAIL_EMAIL_MESSAGE_CLASS='anymail.message.AnymailMessage'
 
     # This replaces django.core.mail.EmailMultiAlternatives
     TEMPLATED_EMAIL_EMAIL_MULTIALTERNATIVES_CLASS='anymail.message.AnymailMessage'
 
 
+Inline images
+==============
+
+You can add inline images to your email using the *InlineImage* class.
+
+First get the image content from a file or a *ImageField*:
+
+.. code-block:: python
+
+    # From a file
+    with open('lena.png', 'rb') as lena:
+      image = lena.read()
+
+    # From a ImageField ('logo' is a ImageField of Company)
+    image = company.logo.read()
```

Comment:
I don't know but would be good make the complete example.
`class Company(...):
 ...
`